### PR TITLE
Ux alert

### PR DESCRIPTION
**Pull request by zac1st1k**

### CODE REVIEW

File: src/components/ux-alert/manifest.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+	"data": {
+		"text": "The text to display on the alert",
+		"class": "[success warning info alert secondary] [round radius]"
+	},
+	"category": "callouts & prompts"
```
Comment (nathf) [inline src/components/ux-alert/manifest.json]: Can these be capitalised? Callouts & Prompts

File: src/components/ux-alert/ux-alert.hbs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div data-alert class="alert-box {{class}}">
+	{{#if isDataModel}}
+		{{{text}}}
+	{{else}}
+		{{yield}}
+	{{/if}}
+	<a href="#" class="close">&times;</a>
```
Comment (nathf) [inline src/components/ux-alert/ux-alert.hbs]: Can't see this close handled anywhere? It should replicate how it works on the docs:
http://foundation.zurb.com/docs/components/alert_boxes.html

File: src/components/ux-alert/ux-alert.feature [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Feature: ux-alert test suite
+
+  Scenario: Loading ux-alert
+    Given I have loaded component "ux-alert" with use case "UseCase"
```
Comment (nathf) [inline src/components/ux-alert/ux-alert.feature]: Create some tests

**Comment by nathf** [comment]
Can you also capitalise all the titles of the use-cases

File: src/components/ux-alert/ux-alert.hbs (hunk 2 above)
Comment (nathf) [inline src/components/ux-alert/ux-alert.hbs]: Maybe you could leverage .teardown()

http://docs.ractivejs.org/latest/ractive-teardown

File: src/components/ux-alert/ux-alert.feature [hunk 4]
```diff
@@ -0,0 +1,6 @@
+Feature: ux-alert test suite
+
+  Scenario: Loading ux-alert
+    Given I have loaded component "ux-alert" with use case "standard"
+    Then the element "div" should be displayed
+    Then the element "div" should have the "className" of "alert-box"
```
Comment (nathf) [inline src/components/ux-alert/ux-alert.feature]: What about a test for clicking the close button?

File: src/components/ux-alert/ux-alert.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Ractive.extend({
+	template: Ractive.defaults.templates['ux-alert'],
+	oninit: function ( options ) {
+		this.on( 'closeClicked', function () {
```
Comment (nathf) [inline src/components/ux-alert/ux-alert.js]: Remove space between bracket and string

File: src/components/ux-alert/ux-alert.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+			this.teardown();
+			return false;
```
Comment (nathf) [inline src/components/ux-alert/ux-alert.js:6]: Q) Do we want to prevent the bubbling of this event? Is `.on('teardown')` enough?

Comment (zac1st1k) [inline src/components/ux-alert/ux-alert.js:6]: A.1) I think the click event should not trigger anything in my opinion.
A.2) I assume so... teardown() will destroy the whole alert component, am I right?

**Pull request merged**